person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1...100)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services
+    return true if id is_of_age? || @parent_permission
```

Comment:
- [ ] it seems in this line you forgot to remove the `id`

Reply:
Thank you for this observation. I have resolved it now.